util/progress/progressui/colors.go

```diff
@@ -0,0 +1,41 @@
+package progressui
+
+import (
+	"os"
+	"strings"
+
+	"github.com/morikuni/aec"
+)
+
+var termColorMap = map[string]aec.ANSI{
+	"": colorRun,
+	"default": aec.DefaultF,
+
+	"black": aec.BlackF,
+	"red": aec.RedF,
+	"green": aec.GreenF,
+	"yellow": aec.YellowF,
+	"blue": aec.BlueF,
+	"magenta": aec.MagentaF,
+	"cyan": aec.CyanF,
+	"white": aec.WhiteF,
+
+	"light-black": aec.LightBlackF,
+	"light-red": aec.LightRedF,
+	"light-green": aec.LightGreenF,
+	"light-yellow": aec.LightYellowF,
+	"light-blue": aec.LightBlueF,
+	"light-magenta": aec.LightMagentaF,
+	"light-cyan": aec.LightCyanF,
+	"light-white": aec.LightWhiteF,
+}
+
+
+func getTermColor() aec.ANSI {
+	envColorString := os.Getenv("BUILDKIT_TERM_COLOR")
```

Comment:
Is there a convention for this? I know `NO_COLOR` is a thing but this looks somewhat weird as there isn't just one color. You are changing the color of the completed item but iirc we have 4 possible different colors used by this progressbar.

Reply:
AFAIC there is not convention on terminal color naming. I just took the ones from "github.com/morikuni/aec" in  lower case.

This PR only affects the "typical buildkit blue" color and not the other 3. It's not a complete theme (I actually like the idea of having it 😺).